### ISSUE TITLE
Support for Aurora Multi-Master Clusters

### ISSUE THREAD

**Issue by bflad** [terraform-providers/terraform-provider-aws#9690]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Announced today: https://aws.amazon.com/about-aws/whats-new/2019/08/amazon-aurora-multimaster-now-generally-available/

### New or Affected Resource(s)

* aws_rds_cluster

### Potential Terraform Configuration

```hcl
resource "aws_rds_cluster" "example" {
  # ... other configuration ...

  engine_mode = "multimaster"
}
```

### References

- https://aws.amazon.com/about-aws/whats-new/2019/08/amazon-aurora-multimaster-now-generally-available/
- https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-multi-master.html

**Comment by bflad** [comment]
Pull request submitted: https://github.com/terraform-providers/terraform-provider-aws/pull/9691

**Comment by bflad** [comment]
Support for this has been merged and will release with version 2.24.0 of the Terraform AWS Provider, in about a week.
